icons/shield_sa_national.svg

```diff
@@ -0,0 +1,56 @@
3 0.1094353,0.272464 0.027472,0.524945 0.2443919,0.760762 0.028435,0.292829 0.064572,0.557238 0.1631924,0.800975 0.044448,0.269543 0.3479729,0.265548 0.6127877,0.263494 0.2374486,-0.101605 0.4304353,-0.109255 0.6999526,-0.04011 0.037881,-0.299693 0.2758696,-0.369578 0.4992786,-0.519641 0.09741,-0.173474 0.26669,-0.103092 0.474031,-0.157764 0.271578,0.0052 0.388464,-0.03311 0.613321,-0.11498 0.171011,0.02804 0.45041,-0.312783 0.647871,-0.386393 0.08291,-0.162003 0.31356,0.01466 0.529536,-0.102586 0.21142,-0.04306 0.406205,-0.257519 0.488138,-0.414741 0.122381,-0.225348 0.416162,-0.227733 0.683429,-0.238288 0.255018,-0.05095 0.385422,-0.274681 0.691106,-0.297328 0.281053,-0.07512 0.557881,-0.184409 0.765337,-0.319293 0.381056,-0.02642 0.143443,-0.527409 0.214511,-0.653441 0.08156,-0.276873 0.57915,-0.300448 0.829043,-0.474832 0.260635,-0.03442 0.644025,-0.439445 0.768162,-0.247728 0.178655,-0.07158 0.465272,0.179615 0.605739,-0.15167 0.225364,-0.09146 -0.0087,-0.544765 0.311861,-0.678317 0.338195,-0.02463 0.506419,0.03713 0.705235,-0.08414 -0.04914,-0.625937 0.461364,-0.825775 0.938729,-0.942253 0.150754,-0.263545 -0.03863,-0.445858 -0.06129,-0.727516 0.124519,-0.16605 -0.02143,-0.586593 0.261629,-0.697469 0.03883,0.444876 0.429151,0.03411 0.392414,-0.12238 -0.0055,-0.273058 0.157486,-0.320926 0.344457,-0.435366 0.07941,-0.259705 0.168815,-0.429732 0.324196,-0.554808 0.04572,-0.243209 0.166439,-0.444061 0.121693,-0.7379902 C 20.83869,9.8127734 20.522218,9.6861652 20.431585,9.3845407 20.225535,9.0634961 19.909041,8.6877844 19.488669,8.7582545 19.208358,8.6333385 18.879523,8.5996632 18.632096,8.3597677 18.409775,8.2029922 18.324512,7.9469118 18.145442,7.7046262 18.228084,7.3904927 17.954016,7.1522435 18.062677,6.8758794 18.13613,6.6397776 18.038083,6.1279116 17.824484,6.4395324 17.65749,6.6516302 17.75338,6.8468004 17.516677,6.9552645 17.343123,7.0368255 17.207302,7.3997479 16.985047,7.508199 16.853309,7.7099414 16.66423,7.9366774 16.625785,8.1185758 16.497557,8.3155226 16.247834,8.1929588 16.019946,8.3246583 15.779121,8.5928635 15.48286,8.2518563 15.197778,8.3180283 15.078781,8.4626691 14.572212,8.6589364 14.655156,8.2633123 14.5005,8.1783003 14.264413,8.2734113 14.3739,7.9603644 14.463139,7.747792 14.561661,7.4824917 14.387665,7.2811401 14.305115,7.0836348 14.526737,6.7783622 14.202765,6.6051108 13.995404,6.4983198 13.783931,6.5730848 13.73848,6.9094551 13.608026,6.9542481 13.706253,7.7807895 13.58612,7.306844 13.498098,7.0060554 13.134601,6.8243725 13.246226,6.4680507 13.259149,6.121132 13.224122,6.1436112 12.98783,5.8459692 12.776669,5.7747115 12.67084,5.5302316 12.496535,5.4115344 12.366755,5.1907696 12.017655,5.0697559 12.000264,4.7897077 11.855838,4.5757747 11.739645,4.2199266 11.364906,4.3218688 10.98494,4.3812401 10.959643,3.7519039 10.549254,3.8545496 9.9411817,3.7641064 9.3036658,3.7462157 8.713742,3.6110542 7.6917476,2.825552 6.7062758,1.9857938 5.5718481,1.3645029 5.2146445,1.0783532 4.7379204,1.1134174 4.3137644,1 Z" id="polyF1S187P4"/>
```

Comment:
If we simplify this shape a bit, I think it would become more feasible to tweak it to better accommodate the numerals at a slightly larger font size. I don’t have any concrete suggestions, since this is a pretty irregular shape, but maybe play around with it a bit.